packages/plugins/plugin-transcription/src/components/Transcript/Transcript.tsx

```diff
@@ -20,26 +21,37 @@ import {
   useTextEditor,
 } from '@dxos/react-ui-editor';
 import { mx } from '@dxos/react-ui-theme';
+import { type MessageType } from '@dxos/schema';
 import { isNotFalsy } from '@dxos/util';
 
 import { transcript } from './transcript-extension';
-import { type BlockModel } from '../../model';
-import { type TranscriptType, type TranscriptBlock } from '../../types';
+import { type SerializationModel } from '../../model';
+import { type TranscriptType } from '../../types';
 
-export const renderMarkdown = (block: TranscriptBlock, debug = false): string[] => {
-  // TODO(burdon): Use link/reference markup for users (with popover).
-  // TODO(burdon): Color and avatar.
-  return [
-    `###### ${block.authorName}` + (debug ? ` (${block.id})` : ''),
-    block.segments.map((segment) => segment.text.trim()).join(' '),
-    '',
-  ];
-};
+export const renderMarkdown =
+  (identities: Identity[]) =>
```

Comment:
i'm adding Start/Stop messages, which will be anonymous. so this might need special handling.